packages/components/src/SubHeaderBar/InputTitleSubHeader/TitleSubHeader.component.js

```diff
@@ -14,27 +14,38 @@ function TitleSubHeader({ title, subTitle, onEdit, t }) {
 					'tc-subheader-details-text-title',
 				)}
 			>
-				<h1
-					onDoubleClick={onEdit}
-					className={classNames(
-						theme['tc-subheader-details-text-title-wording'],
-						'tc-subheader-details-text-title-wording',
-					)}
-				>
-					{title}
-				</h1>
-				<Action
-					name="action-edit-title"
-					label={t('MODIFY_TOOLTIP', { defaultValue: 'Modify' })}
-					icon="talend-pencil"
-					onClick={onEdit}
-					bsStyle="link"
-					className={classNames(
-						theme['tc-subheader-details-text-title-pencil'],
-						'tc-subheader-details-text-title-pencil',
-					)}
-					hideLabel
-				/>
+				{edit ? (
+					<button
+						className={classNames(
+							theme['tc-subheader-details-text-title-wording-button'],
+							'tc-subheader-details-text-title-wording-button', 'btn', 'btn-link'
+						)}
+						onDoubleClick={onEdit}
+					>{title}</button>
+				) : (
+					<h1
+						className={classNames(
+							theme['tc-subheader-details-text-title-wording'],
+							'tc-subheader-details-text-title-wording',
+						)}
+					>
+						{title}
+					</h1>
+				)}
+				{edit && (
+					<Action
+						name="action-edit-title"
+						label={t('MODIFY_TOOLTIP', { defaultValue: 'Modify' })}
```

Comment:
Stil ko : Can you change default to Edit please ?
(This is to unify the way we call edit, UX is aware and should have changed the mockup)